localisation/00_wc_sad.csv

```diff
@@ -265,33 +265,33 @@ EVTOPTB_WCSAD.91;I will seek out Malorne.;Je vais chercher Malorne.;Ich werde Ma
 EVTDESC_WCSAD.112;Despite spending weeks in the wilds you have still failed to create any connection with Malorne. It seems like the Wild God has decided to ignore you.;Bien que vous ayez pass� des semaines dans la nature, vous n'avez toujours pas r�ussi � cr�er un lien avec Malorne. Il semble que le Dieu sauvage ait d�cid� de vous ignorer.;Obwohl Ihr Wochen in der Wildnis verbracht habt, ist es Euch immer noch nicht gelungen, eine Verbindung zu Malorne herzustellen. Es scheint, als h�tte die wilde Gottheit beschlossen, Euch zu ignorieren.;;;;;;;;;;;x
 EVTOPTA_WCSAD.112;I should just give up and try again in a few years.;Je devrais abandonner et r�essayer dans quelques ann�es.;Ich sollte einfach aufgeben und es in ein paar Jahren noch einmal versuchen.;;;;;;;;;;;x
 EVTDESC_WCSAD.120;You've been restless for the whole day, eagerly awaiting the dusk. At last the moment has come, you let out a powerful howl before running off into the wilds.;Vous avez �t� nerveux toute la journ�e, attendant avec impatience le cr�puscule. Le moment est enfin venu, vous laissez �chapper un puissant hurlement avant de vous enfuir dans la nature.;Ihr wart den ganzen Tag �ber unruhig und habt mit Spannung die D�mmerung erwartet. Endlich ist der Moment gekommen, in dem Ihr ein kraftvolles Heulen loslasst, bevor Ihr in die Wildnis davonlauft.;;;;;;;;;;;x
 EVTOPTA_WCSAD.120;The hunt begins.;La chasse commence.;Die Jagd beginnt.;;;;;;;;;;;x
-EVTDESC_WCSAD.121;After some time of hunting you had finally picked up a scent. You follow it to large swath of bushes. Your prey finnaly realizes your presence and jumps out of thicket trying to escape.;Apr�s un certain temps de chasse, vous aviez enfin trouv� une odeur. Vous le suivez jusqu'� une large bande de buissons. Votre proie se rend compte de votre pr�sence et saute hors des fourr�s pour tenter de s'�chapper.;Nach einiger Zeit auf der Jagd habt Ihr schlie�lich eine F�hrte aufgenommen. Ihr folgt ihm bis zu einem gro�en Buschwerk. Eure Beute bemerkt schlie�lich Eure Anwesenheit und springt aus dem Dickicht, um zu entkommen.;;;;;;;;;;;x
+EVTDESC_WCSAD.121;After some time of hunting you had finally picked up a scent. You follow it to large swath of bushes. Your prey finnaly realises your presence and jumps out of thicket trying to escape.;Apr�s un certain temps de chasse, vous aviez enfin trouv� une odeur. Vous le suivez jusqu'� une large bande de buissons. Votre proie se rend compte de votre pr�sence et saute hors des fourr�s pour tenter de s'�chapper.;Nach einiger Zeit auf der Jagd habt Ihr schlie�lich eine F�hrte aufgenommen. Ihr folgt ihm bis zu einem gro�en Buschwerk. Eure Beute bemerkt schlie�lich Eure Anwesenheit und springt aus dem Dickicht, um zu entkommen.;;;;;;;;;;;x
```

Comment:
Made a few edits. First sentence, second sentence ending, finally, leaps out of the thicket.
```suggestion
EVTDESC_WCSAD.121;After hunting for some time you have finally picked up a scent. You follow it to a large swath of trees. Your prey finally realises your presence and leaps out of the thicket trying to escape.;Apr�s un certain temps de chasse, vous aviez enfin trouv� une odeur. Vous le suivez jusqu'� une large bande de buissons. Votre proie se rend compte de votre pr�sence et saute hors des fourr�s pour tenter de s'�chapper.;Nach einiger Zeit auf der Jagd habt Ihr schlie�lich eine F�hrte aufgenommen. Ihr folgt ihm bis zu einem gro�en Buschwerk. Eure Beute bemerkt schlie�lich Eure Anwesenheit und springt aus dem Dickicht, um zu entkommen.;;;;;;;;;;;x
```

Reply:
I'd keep the "swath of bushes" from before, rest is changed.